### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.7][IntelliJ][Wipro]: Keyboard focus is not visible on "Install" button and "Restart IntelliJ IDEA" button in Browse repositories dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1428]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. IntelliJ IDEA start screen appears, Navigate and select  Plugins under Configure combo box.
3. Plugins dialog will open.
4. Navigate to " Browse repositories" button and hit enter.
5. In "Browse repositories" dialog,Select the "azure toolkit for intellij"  plugin and navigate to "Install" button using Ctrl+tab key and hit enter ,then observe the keyboard focus.

**Note:** Same issue is observed for "Restart Intellij IDEA" button after installing the plugin.

**Expected Result:**
Keyboard focus should be  visible on "Install" button and "Restart Intellij IDEA" in Browse repositories dialog.

**Actual Result:**
Keyboard focus is not visible on "Install" button and "Restart Intellij IDEA" in Browse repositories dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![focus is not visible on install button](https://user-images.githubusercontent.com/22956130/35721910-1aad2b90-07a9-11e8-85e1-179c50f8b57d.png)
![no visual focus on restart intellij button](https://user-images.githubusercontent.com/22956130/35721919-20a88134-07a9-11e8-834e-24d4e7095716.png)

**Comment by Eskibear** [comment]
It's the IDE's own dialog, it's out of the toolkit's scope.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749860
